docs/admin/user-management.rst

```diff
@@ -38,8 +38,8 @@ Roles are entities that are **not** allowed to login, but can be assigned
 privileges and they can be granted to other roles, thus creating a role
 hierarchy, or directly to users. For example, a role ``myschema_dql_role`` can
 be granted with ``DQL`` privileges on schema ``myschema`` and afterwards the
-role can be :ref:granted`<grant_roles>` to a user, which will automatically
-inherit those privileges from the ``myschema_dql_role``. A role
+role can be :ref:granted`<roles_inheritance>` to a user, which will
+automatically inherit those privileges from the ``myschema_dql_role``. A role
```

Comment:
Minor: feel like we need to link word "inherit" judging by the contents of the link instead of linking "grant"